util/json_renderer.hpp

```diff
@@ -76,7 +76,7 @@ struct Renderer : mapbox::util::static_visitor<>
     void operator()(const Array &array) const
     {
         out << "[";
-        for (auto it = array.values.cend(), end = array.values.cend(); it != end;)
```

Comment:
@TheMarex I did a git blame and it looks like I accidentally introduced that when doing the global [cache end iterator](https://github.com/Project-OSRM/osrm-backend/pull/1687) thing. Ouch! At least now we know how many people are using the json renderer with the current release. :P